app/models/course/lesson_plan_item.rb

```diff
@@ -0,0 +1,8 @@
+class Course::LessonPlanItem < ActiveRecord::Base
+  actable
+  stampable
+
+  validates_numericality_of :base_exp, only_integer: true
+  validates_numericality_of :time_bonus_exp, only_integer: true
+  validates_numericality_of :extra_bonus_exp, only_integer: true
```

Comment:
Prefer the new style validations `validates :column, numericality: value` over `validates_numericality_of`.